src/renderer.c

```diff
@@ -281,8 +281,10 @@ RenFont* ren_font_load(const char* path, float size, ERenFontAntialiasing antial
     font->underline_thickness = (unsigned short)((face->underline_thickness / (float)face->units_per_EM) * font->size);
   if(!font->underline_thickness) font->underline_thickness = ceil((double) font->height / 14.0);
 
-  if (FT_Load_Char(face, ' ', font_set_load_options(font)))
+  if (FT_Load_Char(face, ' ', font_set_load_options(font))) {
+    free(font);
```

Comment:
A better solution would be using `FT_Done_Face()` and putting it into the body of the `failure` label, like it how the face is supposed to be freed.